jupyter/jupyter-web-app/base/service.yaml

```diff
@@ -7,20 +7,17 @@ metadata:
       apiVersion: ambassador/v0
       kind:  Mapping
       name: webapp_mapping
-      prefix: /jupyter/
-      service: jupyter-web-app.kubeflow
+      prefix: /$(prefix)/
```

Comment:
Are we sure we want to make this configurable? It might be assumed in other places in Kubeflow.
I don't think that needs to block this PR though.